backend/uasmartsignage/src/main/java/deti/uas/uasmartsignage/Models/AppUser.java

```diff
@@ -18,14 +18,14 @@
 @NoArgsConstructor
 @Entity
 @Table(name = "Users")
-public class User{
+public class AppUser{
```

Comment:
Any reason for this change?

Reply:
There is another class from Spring Security called "User"